src/grandorgue/updater/GONewReleaseDialog.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2024 GrandOrgue contributors (see AUTHORS)
```

Comment:
What will the copyright updater precommit hook do in next year? Will it replace with `2025` or with `2024-2025`?

@rousseldenis @larspalo what is our policy for copyright headers in new files? Must have they content 2009 or not?

Reply:
@oleg68 Using a range like 2009-2024 is perhaps best describing the whole software release (the copyright notice according to GNU-GPL is tied to finished releases, not when the code was written). https://www.gnu.org/licenses/gpl-howto.html

Reply:
Yes, the gnu website ended by saying we can decide and explain what to do. IMHO, new files could mention 2024 only and then modifications will change to 2024-2025 for instance (that's the behavior I use on other open source projects), but we can decide to use the same range for the whole files.